src/api-service/__app__/onefuzzlib/azure/vmss.py

```diff
@@ -180,6 +180,11 @@ def reimage_vmss_nodes(name: UUID, vm_ids: Set[UUID]) -> Optional[Error]:
             logging.info("unable to find vm_id for %s:%s", name, vm_id)
 
     if instance_ids:
+        compute_client.virtual_machine_scale_sets.begin_update_instances(
```

Comment:
Does the `reimage` gets queued up if there is an `update` in progress? 
I am concerned that the second operation my fail unless we wait for the result of the first.

Reply:
Adding a comment here for the record. Yes, according to the VMSS team.

Reply:
I had the same question! I suggest we add a comment in the code documenting this expectation (and why we have it).

Reply:
Some logging here might be helpful, too.

Reply:
Cheick and I chatted about this yesterday. I'll make sure to follow-up on the comment anyways next time. 

I'll add a comment _and_ logging, because I agree it'll be helpful down the road.

Reply:
@ranweiler - I added a comment and logging.